packages/rum-core/src/domain/assembly.ts

```diff
@@ -121,6 +122,7 @@ export function startRumAssembly(
           },
           synthetics: syntheticsContext,
           ci_test: ciTestContext,
+          display: getDisplayContext(),
```

Comment:
Getting the display context at the assembly stage is the more straightforward way to implement it.
However, for pre-init calls, it means we won't get the display info of the time the event happens, but at the init time. I think it's good enough though